php/commands/core.php

```diff
@@ -235,6 +259,29 @@ private static function _extract( $tarball, $dest ) {
 		self::_rmdir( dirname( $tempdir ) );
 	}
 
+	private static function _extract_zip( $zipfile, $dest ) {
+		if ( ! class_exists( 'ZipArchive' ) ) {
+			throw new \Exception( 'Extracting a zip file requires PharData' );
```

Comment:
I think we should `WP_CLI::error()` if `ZipArchive` isn't find. The check should actually happen much earlier, to make sure we don't leave a partially downloaded file around.